### ISSUE TITLE
Unable to use System.Text.Json iin Jupyter c# kernel

### ISSUE THREAD

**Issue by viblo** [dotnet/interactive#42]
#### Describe the bug
I just installed the c# jupyter kernel today for trying out. I can not get System.Text.Json namespace to import. If I do 
```using System.Text.Json;``` 
I get this error
 ```(1,19): error CS0234: The type or namespace name 'Json' does not exist in the namespace 'System.Text' (are you missing an assembly reference?)```
I also tried to reference it as a nuget package, with 
```#r "nuget:System.Text.Json"``` 
and it complete successful, but I still cannot use the namespace.

If I run ```Environment.Version``` from within the notebook I get 3.0.0 back. And ```dotnet --version``` shows 3.0.100 so I think I have the proper version installed.

I have installed some other packages, for example MongoDB.Driver could be installed with nuget and imported with using just fine, so it seems its something specifically with this package.

#### Did this error occur while using `dotnet try` or online?
- [x] `dotnet-try`
- [ ] online

#### What kind of error was it?
- [ ] User Interface (UI):  For example the output never displayed 
- [ ] Service Error: For example "The service is temporarily unavailable. We are working on it"
- [x] Other:

#### Screenshots
![image](https://user-images.githubusercontent.com/1195272/68773618-32dea080-062c-11ea-864a-45b42dd6b7a5.png)

#### Please complete the following:
 - OS
    - [x] Windows 10
    - [ ] macOS
    - [ ] Linux (Please specify distro)
    - [ ] iOS
    - [ ] Android
 - Browser 
    - [ ] Chrome
    - [ ] Edge
    - [ ] Safari
    - [x] Firefox

**Comment by osmanzeki** [comment]
I tried this today and seem to have a similar (or same?) issue. In this case, I am running the C# kernel in Jupyter in Windows 10 WSL.

![image](https://user-images.githubusercontent.com/960438/85926047-a86d4900-b86a-11ea-9bbe-bbde8361b73f.png)

**Comment by jonsequitur** [comment]
@osmanzeki Can you run `#!about` in the notebook and post the version here? Thanks.

**Comment by osmanzeki** [comment]
> @osmanzeki Can you run `#!about` in the notebook and post the version here? Thanks.

@jonsequitur Ok, I tried running that command and noticed that nothing was happening at all. Its then that I realized that I had installed the kernels using the command listed in Scott's article:

```
dotnet try jupyter install
```

Which is the one causing issues actually! I just now noticed that one should install using `interactive` in order for things to work properly (as instructed [here](https://nteract.io/kernels/dotnet)):

```
dotnet interactive jupyter install
```

I failed to realize that there were basically two different sets of kernels out there.

I can confirm that everything works fine now on my end with the `interactive` kernels (`System.Text.Json` loads properly) and that I do get the output of `#!about` in that kernel.

![image](https://user-images.githubusercontent.com/960438/86040107-71856780-ba11-11ea-9d95-c61edd5b35ba.png)

As a new user to these, I don't exactly know what the difference is between `dotnet-try` and `dotnet-interactive`, that confusion was the main source of the problem in my case.

Hope this is helpful to someone else playing with these out there. Thanks for the help @jonsequitur !
